### ISSUE TITLE
Keep the original code indentation (Origin: bugzilla #793570)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#6335]
status NEW severity *normal* in component *documentation* for *---*
Reported in version *1.8.15-GIT* on platform *Other*
Assigned to: Dimitri van Heesch

Original attachment names and IDs:
- _[img1.png](https://sourceforge.net/projects/doxygen/files/attachments/bug793570_368530_img1.png/download)_ (ID 368530)
- _[img2.png](https://sourceforge.net/projects/doxygen/files/attachments/bug793570_368531_img2.png/download)_ (ID 368531)

On 2018-02-18 18:53:50 +0000, silvioprog wrote:

    Created attachment 368530
    The img1 picture referenced in the issue
    
    Hi.
    
    I have the following documented code:
    
    
    ```c
    /**
     * Reads a zero-terminated value from string handle @p str.
     * @param[in] str String handle.
     * @param[out] val Value to be read.
     * @param[in,out] len Pointer to specify then store the value length
    including the terminating null byte (<tt>`\0`</tt>).
     * @retval 0 - Success.
     * @retval -EINVAL - Invalid argument.
     * @retval -ENOBUFS - No buffer space available.
     */
    extern int bk_str_read(struct bk_str *str, char *val, size_t *len);
    ```
    
    it generated the attached picture (`img1.png`). But, it should keep the
    original indentation, generating something like the second picture
    (`img2.png`).
    
    Thank you!

On 2018-02-18 18:54:43 +0000, silvioprog wrote:

    Created attachment 368531
    The img2 picture referenced in the issue

On 2018-02-18 18:58:31 +0000, silvioprog wrote:

    > generating something like the second picture (`img2.png`)
    
    Note the last parenthesis ")" after "size_t * len", the breaking line should
    be removed between them.

**Comment by albert-github** [comment]
I think it would make sense to move the closing bracket in case of 2 or more arguments to the end of the previous line as compared to now to the next line. In case on 0 or 1 argument this is already the case:

![image](https://user-images.githubusercontent.com/5223533/203071575-2dd37ea3-02a7-4549-9ac1-ae8d308d28d3.png)

![image](https://user-images.githubusercontent.com/5223533/203071619-c8e0441a-4847-4a06-99e4-f8028ea9a2f1.png)

though not for 2 or more arguments:
![image](https://user-images.githubusercontent.com/5223533/203071726-369908b5-60b3-47a6-be62-75ae08bf7590.png)


@doxygen are there any objections against this?

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10428

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.10.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
